### DISCUSSION TITLE
SRT with continous timing

### DISCUSSION THREAD

**Discussion by himberjack** [openai/whisper#948] [Q&A]
I am trying the Whisper and I noticed that no matter what, the first subtitle starts from 0:00:00 even when the sentence starts after a few secs, and then the next sentence will start where the previous has ended. How can I run the whisper to get the exact timing of the words?

**Comment by SinanAkkoyun** [comment]
Hm, if you need absolute timestamps, they are working on that
But in the meantime here is what you can do:
Use a VAD (voice activity detection) model (perhaps speechbrain) to detect when the sentence begins. Store the time at which it detects the first speech and offset every timestamp from whisper by that amount :)

**Reply by himberjack** [reply]
Thanks! is that available to run through the CLI?

**Reply by SinanAkkoyun** [reply]
No, that would be a custom python implementation.
